docs/azure/sdk/resource-management.md

```diff
@@ -118,7 +144,7 @@ This object provides most of the logical collection operations.
 |-|-|
 | Iterate/List | GetAll() |
 | Index | Get(string name) |
-| Add | CreateOrUpdate(string name, [Resource]Data data) |
+| Add | CreateOrUpdate(Azure.WaitUntil waitUntil, string name, [ResourceName]Data data) |
 | Contains | Exists(string name) |
 
 For most things, the parent will be a **ResourceGroup**. However, each parent-child relationship is represented this way. For example, a **Subnet** is a child of a **VirtualNetwork** and a **ResourceGroup** is a child of a **Subscription**.
```

Comment:
The word "however" here is confusing. Because the first half says "most" cases, combined with the last sentence, I think here should be only still exist special cases? But I don't see a clear transitive relationship.